beacon_node/execution_layer/src/test_utils/mod.rs

```diff
@@ -487,6 +487,7 @@ pub struct StaticNewPayloadResponse {
     should_import: bool,
 }
 #[derive(Debug)]
+// We don't use the string value directly, but it's used in the Debug impl which is required by `warp::reject::Reject`.
```

Comment:
don't think we need this comment because of the `e.0` below

Reply:
oh yeah I removed the `allow` attribute but forgot to remove the comment. I'll fix!

Reply:
https://github.com/sigp/lighthouse/pull/5180/commits/3b1da953e9d61d674348e30fd3838e4ffc9d19bf